### DISCUSSION TITLE
3D Map Configuration Names are hidden

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#1012] [General]
Since I've used MT (18.X?), when I've wanted to configure the 3D map configuration names, I've always seen them as "encrypted"/hidden

is it normal ? or is it a bug ?

![image](https://user-images.githubusercontent.com/25235665/224070446-81de3835-1b8e-460b-84b7-71ad4c899a75.png)

**Comment by wolfgang-ch** [comment]
The dots are normal, I had no idea how to name them 😄

**Comment by wolfgang-ch** [comment]
In the `Name` field you can rename it

![image](https://user-images.githubusercontent.com/1283445/224080505-066b8acd-18ca-46a5-9dc6-cdf719532dcf.png)
